js/app/demo/rulesConfig.jsx

```diff
@@ -26,12 +27,24 @@ define(['react'], function(React) {
         function shouldBeChecked(rule) {
             return props.config[rule] && props.config[rule] !== "off" && props.config[rule] !== 0;
         }
+        function handleChange(e, key) {
+            var updatedConfig = Object.assign({}, props.config);
+
+            if (e.target.checked) {
+                updatedConfig[key] = "error";
+            } else {
+                delete updatedConfig[key];
+            }
+            props.onUpdate(updatedConfig);
+        }
         function getRow(i) {
             var limit = Math.ceil(props.ruleNames.length / 3);
-            const start = limit * i;
```

Comment:
As a result of this error, the demo was unusable on non-ES6 browsers for a few days. :confused: